package nextstep.member.application.exception;

public class AuthException extends RuntimeException {


}
